Unix/tools/auth_tests_setup.sh

```diff
@@ -37,34 +40,60 @@ do
 done
 
 #
-# if required env variables don't exist, then quit
+# Setup the NTLM auth file for client and server. They must be different files because 
+# if the differeing permissions, as the server file must be owned by root and the client by
+# the user.
+#
+# If there is already a cred file present we defer to it. In configure we possibly copied over
+# a prebuilt cred file for the server from Unix/etc/creds. If such a file exists, it must contain the test
+# credentials.
```

Comment:
Oh, this is in test code. Test code requires --dev. Just make that clear here by stating it.